stubs/taint/core_privacy_security/django_sources_sinks.pysa

```diff
@@ -1,52 +1,43 @@
+# Sources and Sinks for Django 3.2 LTS released on April, 2021.
+
 # HttpRequest sources
 
 django.http.request.HttpRequest.COOKIES: TaintSource[UserControlled, UserControlled_Meta] = ...
 django.http.request.HttpRequest.META: TaintSource[UserControlled, UserControlled_Meta] = ...
-django.http.request.HttpRequest.user_agent_string: TaintSource[UserControlled, UserControlled_Meta] = ...
-django.http.request.HttpRequest.user_agent: TaintSource[UserControlled, UserControlled_Meta] = ...
-
 django.http.request.HttpRequest.FILES: TaintSource[UserControlled, UserControlled_Payload, Via[request_files]] = ...
 django.http.request.HttpRequest.GET: TaintSource[UserControlled, UserControlled_Payload] = ...
 django.http.request.HttpRequest.POST: TaintSource[UserControlled, UserControlled_Payload] = ...
-django.http.request.HttpRequest.REQUEST: TaintSource[UserControlled, UserControlled_Payload] = ...
-django.http.request.HttpRequest.body: TaintSource[UserControlled, UserControlled_Payload] = ...
-
+def django.http.request.HttpRequest.body(self) -> TaintSource[UserControlled, UserControlled_Payload]: ...
+django.http.request.HttpRequest.content_type: TaintSource[UserControlled, UserControlled_Payload]
+django.http.request.HttpRequest.content_params: TaintSource[UserControlled, UserControlled_Payload]
+def django.http.request.HttpRequest.accepted_types(self) -> TaintSource[UserControlled, UserControlled_Payload]: ...
+def django.http.request.HttpRequest.encoding(self) -> TaintSource[UserControlled, UserControlled_Payload]: ...
+def django.http.request.HttpRequest.headers(self) -> TaintSource[UserControlled, UserControlled_Payload]: ...
 django.http.request.HttpRequest.GET: Sanitize = ...
 django.http.request.HttpRequest.POST: Sanitize = ...
 
 django.http.request.HttpRequest.COOKIES: TaintSource[Cookies] = ...
 def django.http.request.HttpRequest.get_signed_cookie(self, key, default, salt, max_age) -> TaintSource[Cookies]: ...
-django.http.response.HttpResponseBase.cookies: TaintSource[Cookies] = ...
-
 
 django.http.request.HttpRequest.META: TaintSource[HeaderData] = ...
 def django.http.request.HttpRequest.__repr__(self) -> TaintSource[HeaderData]: ...
 django.core.handlers.wsgi.WSGIRequest.environ: TaintSource[HeaderData]
-def django.http.request.build_request_repr(
-  request, path_override,
-  GET_override,
-  POST_override,
-  COOKIES_override,
-  META_override
-) -> TaintSource[HeaderData]: ...
-
-def django.http.request.HttpRequest.get_full_path(self) -> TaintSource[URL]: ...
+
+def django.http.request.HttpRequest.get_full_path(self, force_append_slash) -> TaintSource[URL]: ...
```

Comment:
Here and other places: you don't need to specify parameters if you are not specifying sinks on them.
```suggestion
def django.http.request.HttpRequest.get_full_path() -> TaintSource[URL]: ...
```

Reply:
This was just for readability, to emphasize that the function has parameters. Can we keep the parameters explicitly specified in the .pysa file? Just let me know. If you tell, I will remove the parameters that would not be ever needed to be "tainted", like this boolean one.

Reply:
Not specifying non-sink parameters in Pysa models is better for maintainability. If for example, the API changes and `force_append_slash` gets renamed to something else, your model here will break if you specified explicit parameter names, while it would just continue to work if you didn't specify the parameter name. 

It's better to not add additional maintaining work for yourself, since keeping the parameters up-to-date is not really relevant to the main idea we are trying to capture with this model, which is that the `get_full_path()` function returns a tainted value (regardless of what parameters are passed in).

Reply:
@onionymous -- Thank you very much! I will change this now.